x-pack/plugins/ml/public/components/tooltip/index.js

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import './tooltip_directive';
```

Comment:
new line missing here.
not sure if this is a bad thing as there is no lint rule for it. but github flags it up.
`"files.insertFinalNewline": true` in vscode will do this automatically.
The same for `x-pack/plugins/ml/public/components/tooltip/tooltip_directive.js`